portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase

```diff
@@ -25,6 +25,86 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-184580 - StaleObjectExcpetion when changing a web content portlet's displayed article in a publication."
+	@priority = 4
+	test CanChangeWebContentWidgetDisplayedArticleInPublication {
+		task ("Create two Web Contents and two Content Pages") {
```

Comment:
The last step is to assert that there is no error thrown, so we need to add the propery of `property test.assert.warning.exceptions = "true";` for the test.